circle-mlir/models/gen_models.py

```diff
@@ -155,6 +159,13 @@ def generate_models(models):
 
         models = models_unit + models_net
 
-        generate_models(models)
+        # these items are not supported in ai-edge-torch as of editing this file.
+        excludes = [
+            'Cast_F32_R4_U8', 'ConvTranspose2d_F32_R4_op01', 'QuantizeLinear_F32_R3_i16',
+            'QuantizeLinear_F32_R3_i16_cw', 'QuantizeLinear_F32_R3_ui8',
+            'QuantizeLinear_F32_R3_ui8_fq', 'QuantizeLinear_F32_R4_i16_cw',
+            'QuantizeLinear_F32_R4_ui8_cw', 'Slice_F32_R3_unk_1', 'Slice_F32_R3_unk_2'
+        ]
+        generate_models(models, excludes)
     else:
-        generate_models(models)
+        generate_models(models, None)
```

Comment:
we may need to check the models event for excluded list for individual model testing,
to check the model is working for new ai-edge-torch package.